koku/masu/celery/tasks.py

```diff
@@ -259,6 +270,12 @@ def sync_data_to_customer(dump_request_uuid):
         dump_request.status = DataExportRequest.ERROR
         dump_request.save()
         return
-
+    except SyncedFileInColdStorageError as e:
+        LOG.info(
+            f'One of the request files is currently in cold storage, DataExportRequest'
+            f'{dump_request.uuid}. This task will automatically retry.')
+        dump_request.status = DataExportRequest.ERROR
```

Comment:
I don't think we should set the `status` to `ERROR` since the process is actually working normally despite the interruption. I typically think of error states as terminal states. Maybe we should add a new status here like `RETRYING` or `WAITING` to indicate that things are still going to be processed?